packages/medusa/src/migrations/1680857773273-add-table-product-shipping-profile.ts

```diff
@@ -32,7 +32,7 @@ export class addTableProductShippingProfile1680857773273
         DROP INDEX IF EXISTS "idx_product_shipping_profile_product_id";
         DROP INDEX IF EXISTS "idx_product_shipping_profile_profile_id";
 
-        ALTER TABLE "product" ADD COLUMN IF NOT EXISTS "profile_id";
+        ALTER TABLE "product" ADD COLUMN IF NOT EXISTS "profile_id" CHARACTER VARYING;
```

Comment:
comment: I don't know if this is the right approach. The migration has been run for many users already. If the current type is causing issues, I'd rather have us create a new migration to fix it. What do you think?

Reply:
Ah got it. I failed to see it was in the down function. Should be OK then 👍